src/contributors/03-docker-development.md

```diff
@@ -23,6 +23,67 @@ Get the code with submodules:
 git clone https://github.com/LemmyNet/lemmy --recursive
 ```
 
+### Building
+
+Use these commands to create a custom container based on your local branch and tagged accordingly.
+
+This is useful if you want to modify the source code of your instance to add some extra functionalities which are not available in the main release.
+
+```bash
+sudo docker build . -f docker/Dockerfile -t "lemmy:${git rev-parse --abbrev-ref HEAD}"
+```
+
+#### Build Troubleshooting
+
+In case the build fails, the following might help resolve it
+
+##### Translations missing
+
+If you see an error like this
+```
+Error: FileRead { file: "translations/email/en.json", source: Os { code: 2, kind: NotFound, message: "No such file or directory" } }
```

Comment:
This is already handled by `git clone --recursive` above. Anyway you can move it right under that command so its all in one place.